api/services/gameService.js

```diff
@@ -55,6 +55,8 @@ function tempGame(game, p0, p1) {
   this.passes = game.passes;
   this.resolving = game.resolving;
   this.lastEvent = game.lastEvent;
+  this.result = game.result;
+  this.ranked = game.ranked;
```

Comment:
Adding these to the return value of `.populateGame()` so they can be read off it when we pass a populated game as an arg to the new helper

Reply:
I wonder if it's worth refactoring this to assign them all in a loop instead of one by one?
```
Object.entries(game).forEach(([key, val]) => {
    if (key === 'players') {
        this[key] = [p0, p1];
        return;
    }
    this[key] = val;
});
```

Reply:
Yes I think so, but in the name of scope let's not now. Part of why it was set up this way was to allow deliberate omission of fields from the data sent to users in the socket payload, but that's more important for the user objects (e.g. scrubbing `encryptedPassword`) than for the game

Reply:
Created issue #111 to refactor `populateGame`